src/robusta/cli/self_host.py

```diff
@@ -159,8 +174,8 @@ def gen_config(
     values = SelfHostValues(
         PROVIDER=provider,
         DOMAIN=domain,
-        SITE_URL=f"https://platform.{domain}",
-        PUBLIC_REST_URL=f"https://db.{domain}/rest/v1/",
+        SITE_URL=host_for_params(platform_endpoint_prefix, domain),
```

Comment:
Can we assign a single variable for this and other similar function calls?

Reply:
can you explain?